api/model.go

```diff
@@ -889,7 +902,7 @@ func toVolumeResource(v *longhorn.Volume, ves []*longhorn.Engine, vrs []*longhor
 		RevisionCounterDisabled: v.Spec.RevisionCounterDisabled,
 		Ready:                   ready,
 
-		Share:         v.Spec.Share,
+		AccessMode:    v.Spec.AccessMode,
```

Comment:
If you decide not to update field `v.Spec.AccessMode` for existing volumes, you may still need to make sure this field exposed to UI is not empty.